### PR TITLE
some sd package fixes

### PR DESCRIPTION
**Pull request by brandondutra**

### CODE REVIEW

File: solutionbox/structured_data/mltoolbox/_structured_data/_package.py [hunk 1]
```diff
@@ -86,13 +85,16 @@ def _package_to_staging(staging_package_url):
...
     package_root = os.path.abspath(
-        os.path.join(os.path.dirname(__file__), '../'))
+        os.path.join(os.path.dirname(__file__), '../../'))
     setup_path = os.path.abspath(
         os.path.join(os.path.dirname(__file__), 'master_setup.py'))
     tar_gz_path = os.path.join(staging_package_url, 'staging', 'sd.tar.gz')
 
+    print(__file__)
+    print('proot', package_root)
+    print('setup_path', setup_path)
```
Comment (qimingj) [inline solutionbox/structured_data/mltoolbox/_structured_data/_package.py]: Are there needed?

File: solutionbox/structured_data/mltoolbox/_structured_data/master_setup.py [hunk 2]
```diff
@@ -60,7 +76,7 @@ def get_version():
     'tensorflow==1.0',
     'protobuf==3.1.0',
     'google-cloud-dataflow==0.5.5',
-    #'psutil>=4.3.0'
+    'psutil'
```
Comment (qimingj) [inline solutionbox/structured_data/mltoolbox/_structured_data/master_setup.py]: I thought you had to remove it?

File: solutionbox/structured_data/mltoolbox/_structured_data/_package.py (hunk 1 above)
Comment (brandondutra) [inline solutionbox/structured_data/mltoolbox/_structured_data/_package.py]: nope!

File: solutionbox/structured_data/mltoolbox/_structured_data/master_setup.py (hunk 2 above)
Comment (brandondutra) [inline solutionbox/structured_data/mltoolbox/_structured_data/master_setup.py]: ah, I did, and when I built it copied the file over. But I forgot to re-build it.

**Pull request merged**